client/web/assets.go

```diff
@@ -27,24 +27,31 @@ import (
 //go:embed yarn.lock index.html *.js *.json src/*
 var _ embed.FS
 
-//go:embed build/*
-var embeddedFS embed.FS
-
-// staticfiles serves static files from the build directory.
-var staticfiles http.Handler
-
-func init() {
-	buildFiles := must.Get(fs.Sub(embeddedFS, "build"))
-	staticfiles = http.FileServer(http.FS(buildFiles))
-}
+// defaultIndex is the default index page to serve if web client assets haven't been built.
+// We embed this separately rather than checking in a default build/index.html file
+// to prevent "dirty" builds when build/index.html is overwritten by vite.
```

Comment:
```suggestion
// We embed this separately rather than checking a default build/index.html file
// into git to prevent "dirty" builds when build/index.html is overwritten by Vite.
```